roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -15,20 +15,29 @@ class DockerImageAvailability(OpenShiftCheck):
 
     skopeo_image = "openshift/openshift-ansible"
 
-    # FIXME(juanvallejo): we should consider other possible values of
-    # `deployment_type` (the key here). See
-    # https://github.com/openshift/openshift-ansible/blob/8e26f8c/roles/openshift_repos/vars/main.yml#L7
-    docker_image_base = {
+    deployment_image_info = {
```

Comment:
hm, in the case of a flat list here, we would need to handle cases for images with qualified image tags, non-qualified image tags (`vX.Y` instead of `vX.Y.Z`), and of those two, cases where images exist for one deployment but not for another depending on the install type (only case so far is the `registry-console` image not existing for `origin` deployments in non-containerized installs, and `openshift/origin:v...` existing in containerized installs, but being formatted as `openshift3/ose:v...` for the enterprise deployment). I was trying to avoid returning multiple lists under different conditions in these methods, but will look at this once more and try to simplify further